cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -229,7 +229,9 @@ func initializeWatch(_ *appconfig.CompletedConfig, cloudConfigPath string) (watc
 			case err := <-watch.Errors:
 				klog.Warningf("watcher receives err: %v\n", err)
 			case event := <-watch.Events:
-				klog.Fatalf("config map %s has been updated, restarting pod, received event %v\n", cloudConfigPath, event)
+				if event.Op != fsnotify.Chmod {
```

Comment:
Will CPI change mod now? Did you run into any issue and that's why you changed it?

Reply:
Yes, if not having this line, the CPI pod will keep crashing because of the Chmod events at the config map.

Reply:
Who will trigger the chmod event?

Reply:
My guess is some controller from helm. But I am not really sure about this